demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -0,0 +1,837 @@
+        if command.get('root_object'):
+            function = function.replace('$OUTPUTS$', f"response.get('{command['root_object']}')")
+        else:
+            function = function.replace('$OUTPUTS$', 'response')
+        if command['unique_key']:
+            function = function.replace('$UNIQUEKEY$', command['unique_key'])
+        else:
+            function = function.replace('$UNIQUEKEY$', '')
+        function = function.replace('$CONTEXTNAME$', context_name)
+        return function, req_function
+
+    def get_yaml(self, no_code):
+        # Create the commands section
+        commands = []
+        for command in self.configuration['commands']:
+            args = []
+            for arg in command['arguments']:
+                options = None
+                auto = None
+                arg_name = arg['name']
+                if arg.get('ref'):
+                    arg_name = f"{arg['ref']}_{arg_name}".lower()
+                required = True if arg['required'] else False
+                description = arg.get('description', None)
+                is_array = True if arg['type'] == 'array' else False
+                if description:
+                    description = self.clean_description(description)
+                else:
+                    description = ''
+                if arg['properties']:
+                    for k, v in arg['properties'].items():
+                        prop_arg_name = f'{arg_name}_{k}'
+                        if description:
+                            prop_description = f'{description} - {k}'
+                        else:
+                            prop_description = f'{arg_name} {k}'
+                        prop_required = True if required is True and v.get('required') else False
+                        options = None
+                        auto = None
+                        if v.get('options'):
+                            auto = 'PREDEFINED'
+                            options = v['options']
+                        args.append(XSOARIntegration.Script.Command.Argument(prop_arg_name, prop_description,
+                                                                             prop_required, auto, options))
+                else:
+                    if arg['options']:
+                        auto = 'PREDEFINED'
+                        options = arg['options']
+
+                    args.append(XSOARIntegration.Script.Command.Argument(arg_name, description, required,
+                                                                         auto, options, is_array))
+
+            outputs = []
+            context_path = command['context_path']
+            for output in command['outputs']:
+                output_name = output['name']
+                if context_path:
+                    output_name = f'{context_path}.{output_name}'
+                if self.context_path:
+                    output_name = f'{self.context_path}.{output_name}'

```

Comment:
is that intentional that we override the output path?

Reply:
why override?

Reply:
because if `self.context_path` is set, `output_name` will always be set in this condition and override whatever was set in the previous condition

Reply:
`output_name` is used in the second condition, it adds to that.